packages/mermaid/src/diagrams/class/parser/classDiagram.jison

```diff
@@ -200,7 +200,7 @@ Function arguments are optional: 'call <callback_name>()' simply executes 'callb
 
 start
     : mermaidDoc
-    | statments
+    | statements
```

Comment:
@jgreywolf @knsv this error comes when building jison. 

But I have no clue how this used to work when it was `statments` 🤷‍♂️ 

```
Conflict in grammar: multiple actions possible when lookahead token is $end in state 4
- reduce by rule: statement -> direction
- reduce by rule: start -> direction
Conflict in grammar: multiple actions possible when lookahead token is NEWLINE in state 4
- reduce by rule: statement -> direction
- reduce by rule: start -> direction

States with conflicts:
State 4
  start -> direction . #lookaheads= $end NEWLINE
  statement -> direction . #lookaheads= $end NEWLINE
```

Reply:
I have added [a fix](https://github.com/mermaid-js/mermaid/pull/4338), can you verify? @knsv @jgreywolf